### ISSUE TITLE
Adding the constant wp_cache in your wp-config.php

### ISSUE THREAD

**Issue by juan790** [Automattic/wp-super-cache#202]
The WP_CACHE line must be moved over.

What’s the problem? How can I solve it?

Thanks.

**Comment by kraftbj** [comment]
Howdy,

I'm not sure I understand the question/issue. Can you rephrase?

**Comment by donnchawp** [comment]
I think this is a support issue if I'm understanding it. 

@juan790 - you should move the line containing the WP_CACHE define UP near the top of the wp-config.php, but also this should be handled in the support forum on wordpress.org.

**Comment by juan790** [comment]
Yes, I'm asking for help and support for this problem.
Sometimes, this message appears in wp-admin of wordpress.

This is the error: _WP_CACHE constant added to wp-config.php If you continue to see this warning message please see point 5 of the http://wordpress.org/extend/plugins/wp-super-cache/faq. The WP_CACHE line must be moved up._

The wordpress support forum seems dead. Here we offer support?

**Comment by donnchawp** [comment]
No, this isn't a support forum, sorry. See my answer above. You need to move the define() in wp-config.php that sets up WP_CACHE up a couple of lines in the file.

**Comment by juan790** [comment]
Ok sorry. Thanks for the support.
